querybook/server/logic/elasticsearch.py

```diff
@@ -78,13 +80,19 @@ def get_hosted_es():
     return hosted_es
 
 
+def _get_partial_dict(field_to_generator, fields=None):
```

Comment:
def _get_dict_by_field(field_to_getter: Dict[str, Any] , fields: Optional[List[str]] =None):

also can you add documentation for this function